rules/prefer_to_be_undefined.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = context => {
+  return {
+    CallExpression(node) {
+      const calleeName = node.callee.name;
+
+      if (
+        calleeName === 'expect' &&
+        node.arguments.length == 1 &&
+        node.parent &&
+        node.parent.type === 'MemberExpression' &&
+        node.parent.parent
+      ) {
+        const parentProperty = node.parent.property;
+        const propertyName = parentProperty.name;
+        const argument = node.parent.parent.arguments[0];
+
+        if (propertyName === 'toBe' && argument.value === undefined) {
```

Comment:
`toEqual` as well

Should probably pull alot of this shared code out into a util